irods_catalog_provider/install_transfer_totals.sh

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash -e
+
+######
+# 1. create new db table
+
+PGPASSWORD=testpassword psql ICAT -h irods-catalog -c "\
+  CREATE TABLE r_transfer_totals ( \
+  user_id BIGINT NOT NULL, \
+  action VARCHAR(250) NOT NULL, \
+  exbibytes BIGINT NOT NULL, \
+  bytes BIGINT NOT NULL, \
+  CONSTRAINT unique_user_action UNIQUE (user_id, action) \
+) \
+"
```

Comment:
In the future, we might consider pulling out this bit so that we can re-use the rest of the script in both provider and consumer (if indeed the rest is the same).